src/openrct2/ride/Vehicle.h

```diff
@@ -282,6 +284,9 @@ enum {
     VEHICLE_ENTRY_ANIMATION_MULTI_DIM_COASTER
 };
 
+#define ANIMATION_SPEED_MODIFIER_COEFFICIENT 32
```

Comment:
All new code should use the C++ counterpart: constexpr.

```suggestion
constexpr const uint8 ANIMATION_SPEED_MODIFIER_COEFFICIENT = 32;
```